src/reducers/tasksReducer.js

```diff
@@ -32,6 +32,24 @@ export default function (state = initialState, action) {
     case 'POST_TASK_REJECTED':
       return state;
 
+    case 'DELETE_TASK_PENDING':
+      return state;
+
+    case 'DELETE_TASK_REJECTED':
+      return state;
+
+    case 'DELETE_TASK_FULFILLED':
+      const newState = JSON.parse(JSON.stringify(state));
```

Comment:
@ajvb You could do `const newState = Object.assign({}, state)` to make a new object based on `state`, which is what it looks like you're trying to do?

@MartyTheeMartian @Moe-Shoman @spetastian Advice?

Reply:
That is correct. Instead of parsing to and from json we can use object.assign to create a new state object. It seems as though the state is not changed at all here. Is that intentional? If aos

Reply:
... If so you can just do `return state`

Reply:
Nvm, just saw the rest of the code in that case clause. But still, you can use object.assign instead.

Reply:
Or ... we can use ES7 spread operator:  `const newState = { ...state };`

Reply:
@martytheemartian Yes, good catch! Since the code is being transpiled we should use spread and rest operators.

Reply:
Spread operator :+1:. Let's do it.